kinetic/distribution.yaml

```diff
@@ -4633,6 +4633,17 @@ repositories:
       type: git
       url: https://github.com/inomuh/indoor_localization.git
       version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/inomuh/indoor_localization-release.git
+      version: 0.0.1-0
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/inomuh/indoor_localization.git
+      version: 0.1.0
```

Comment:
Looks like [CI failed with](https://travis-ci.org/ros/rosdistro/jobs/499814333)

```
verifying diff of file '/home/travis/build/ros/rosdistro/kinetic/distribution.yaml'
No ROSGHPRB_TOKEN set, continuing without checking hooks
ERROR: In file '''/home/travis/build/ros/rosdistro/kinetic/distribution.yaml''': Could not validate source entry for repo indoor_localization with error [[[Could not validate repository with url https://github.com/inomuh/indoor_localization.git and version 0.1.0 from entry at line 4641. Error reason: No branch found matching 0.1.0]]]
```

Did you mean [kinetic-devel](https://github.com/inomuh/indoor_localization/tree/kinetic-devel)?